exercises/06.1-Loop-Array/README.es.md

```diff
@@ -10,6 +10,6 @@ El código ahora imprime el primer elemento en la consola.
 
 ## 💡Pista:
 
-+ Recuerde que para acceder al valor de una posición debe usar el índice `console.log (myArray [index]);`.
++ Recuerda que para acceder al valor de una posición debe usar el índice `console.log (myArray [index]);`.
```

Comment:
`console.log (myArray [index]);` the console.log() could be confusing for the student. Use just this instead: `myArray[index]`